ext/random/randomizer.c

```diff
@@ -278,8 +278,8 @@ PHP_METHOD(Random_Randomizer, getBytes)
 		Z_PARAM_LONG(length)
 	ZEND_PARSE_PARAMETERS_END();
 
-	if (length < 1) {
-		zend_argument_value_error(1, "must be greater than 0");
+	if (length < 0) {
+		zend_argument_value_error(1, "must be greater than or equal to 0");
```

Comment:
Please also adjust the test for `Randomizer::getBytes()`: https://github.com/php/php-src/blob/master/ext/random/tests/03_randomizer/methods/getBytes.phpt

Reply:
done.